src/UIComponents/ShadowTextbox.tsx

```diff
@@ -0,0 +1,48 @@
+import "./ShadowTextbox.css";
+
+export type ShadowTextboxProps = {
+  onKeyDown?: (event: React.KeyboardEvent) => void;
+  onFocus?: (event: React.FocusEvent) => void;
+  onChange: (str: string) => void;
+  text?: string;
+  shadowText?: string;
+  showSpinner?: boolean;
+  placeHolderText?: string;
+};
+
+export const ShadowTextbox = ({
+  onKeyDown,
+  onFocus,
+  onChange,
+  text,
+  shadowText,
+  showSpinner = false,
+  placeHolderText,
+}: ShadowTextboxProps) => {
+  return (
+    <div className="shadowTextbox">
+      {shadowText && (
+        <input
+          className="text shadowText"
+          tabIndex={-1}
+          type="text"
+          value={shadowText || ""}
+          readOnly={true}
+          onChange={() => {}}
+        ></input>
```

Comment:
Does the shadow input need an onChange handler if it's just presentation? (this isn't a critique. I'm fine if it's there to be explicit. I'm just curious)

Reply:
The shadow text box is just presentation so does not need a onchange handler

Reply:
I really like this implementation though. Going to pinch it.

Reply:
I didn't realise the shadow text exists in typeahead